### DISCUSSION TITLE
configure Trusted Headers in the Docker Container

### DISCUSSION THREAD

**Discussion by karazonanas** [shyim/shopware#35] [Q&A]
Hello everyone, I've a question which is really confusing me at the last time.
I'm trying to configure the Trusted Headers since I both use Cloudflare as a DNS Server and Traefik as reverse proxy with this Docker container.
I believe I need to edit the [shopware.yaml](https://github.com/shyim/shopware/blob/master/rootfs/var/www/html/config/packages/shopware.yaml) file, and maybe add something like trusted_headers tag, am I in the correct file or is there any other way to configure the trusted headers?

**Comment by shyim** [comment]
It works by setting TRUSTED_PROXIES environment variable. I guess you could set it to `REMOTE_ADDR` as all requests are trusted from traefik.

**Reply by karazonanas** [reply]
it didn't help, unfortunately. I still get the local IP
when I activate the Maintenance mode, I can't whitelist my IP.
